### PR TITLE
feat: add Unsafe HTML shortcode

### PR DESCRIPTION
**Pull request by amuraru**
## Description

New short-code to include verbatim html code.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shah** [comment]
This looks good to me, thanks!

File: layouts/shortcodes/unsafe-html.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+NOTICE: Adobe permits you to use, modify, and distribute this file in
+accordance with the terms of the Adobe license agreement accompanying
+it. If you have received this file from a source other than Adobe,
+then your use, modification, or distribution of it requires the prior
+written permission of Adobe.
+
+*/}}
+
+<!-- pass HTML as-is (raw) -->
+{{ .Inner }}
```
Comment (alexander-falca) [inline layouts/shortcodes/unsafe-html.html:15]: should we add a small example of usage?

Comment (shah) [reply layouts/shortcodes/unsafe-html.html:15]: Great idea @alexander-falca, an example of using `unsafe-html` shortcode to use a direct `<coral-*>` HTML Markup would be perfect.

**Comment by shah** [comment]
Hi @amuraru I was curious when this will be merged. thanks!

File: layouts/shortcodes/unsafe-html.html (hunk 1 above)
Comment (amuraru) [reply layouts/shortcodes/unsafe-html.html:15]: done

**Pull request merged**